common/display/display.cpp

```diff
@@ -164,8 +164,18 @@ bool Display::GetActiveConfig(uint32_t *config) {
   return true;
 }
 
-bool Display::SetDpmsMode(uint32_t dpms_mode) {
-  return display_queue_->SetDpmsMode(dpms_mode);
```

Comment:
Reverted still the device is not going to the idle state and the screen is still on